src/main/java/net/mcreator/ui/minecraft/BiomeListField.java

```diff
@@ -22,20 +23,44 @@
 import net.mcreator.minecraft.ElementUtil;
 import net.mcreator.ui.MCreator;
 import net.mcreator.ui.component.JItemListField;
+import net.mcreator.ui.dialogs.AddTagDialog;
 import net.mcreator.ui.dialogs.DataListSelectorDialog;
 import net.mcreator.ui.init.L10N;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class BiomeListField extends JItemListField<BiomeEntry> {
 
 	public BiomeListField(MCreator mcreator) {
-		super(mcreator);
+		super(mcreator, false);
+	}
+
+	public BiomeListField(MCreator mcreator, boolean allowTags) {
+		super(mcreator, false, allowTags);
 	}
 
 	@Override protected List<BiomeEntry> getElementsToAdd() {
 		return DataListSelectorDialog.openMultiSelectorDialog(mcreator, ElementUtil::loadAllBiomes,
 						L10N.t("dialog.list_field.biome_list_title"), L10N.t("dialog.list_field.biome_list_message")).stream()
 				.map(e -> new BiomeEntry(mcreator.getWorkspace(), e)).toList();
 	}
+
+	@Override protected List<BiomeEntry> getTagsToAdd() {
+		List<BiomeEntry> tags = new ArrayList<>();
+
+		String tag = AddTagDialog.openAddTagDialog(mcreator, "is_overworld", "is_nether", "is_end", "is_badlands",
+				"is_beach", "is_cave", "is_cold/end", "is_cold/overworld", "is_cold", "is_coniferous", "is_deep_ocean",
+				"is_dense/overworld", "is_dense", "is_desert", "is_dry/end", "is_dry/nether", "is_dry/overworld",
+				"is_dry", "is_end", "is_forest", "is_hill", "is_hot/nether", "is_hot/overworld", "is_hot", "is_jungle",
+				"is_lush", "is_mountain", "is_mountain", "is_mushroom", "is_nether", "is_ocean", "is_overworld",

```

Comment:
`is_mountain` appears twice. Also tags added by Forge (like `is_coniferous`) should have a `forge:` namespace, as otherwise it defaults to `minecraft:`